packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -474,7 +486,7 @@ export interface ContextuallyTypedNodeDataObject {
 	/**
 	 * Fields of this node, indexed by their field keys.
 	 *
-	 * Allow explicit undefined for compatibility with FlexTree, and type-safety on read.
+	 * Allow explicit undefined for compatibility with EditableTree, and type-safety on read.
```

Comment:
I think you accident reverted a recent fix correcting use of EditableTree. Might want to look for anything else that might have gotten merged bad. Maybe making a fresh minimal PR to split out the leaf schema would help mitigate that risk as it should be real easy to aprove and merge quickly

Reply:
Ah, yeah that could be. I'll do an audit pass.

Reply:
leaf schema refactor PR: #20289

Reply:
```suggestion
	 * Allow explicit undefined for compatibility with FlexTree, and type-safety on read.
```